package.json

```
@@ -53,22 +52,31 @@
   "devDependencies": {
     "@ledgerhq/hw-transport-node-speculos": "^6.27.10",
     "@types/argparse": "^2.0.10",
-    "@types/mocha": "^8.2.1",
-    "@typescript-eslint/eslint-plugin": "^5.17.0",
-    "@typescript-eslint/parser": "^5.17.0",
-    "eslint": "^8.12.0",
-    "eslint-config-airbnb-base": "^15.0.0",
-    "eslint-plugin-import": "^2.27.5",
+    "@types/mocha": "^10.0.0",
+    "@types/node": "^18.12.0",
+    "@typescript-eslint/eslint-plugin": "^5.42.1",
+    "@typescript-eslint/parser": "^5.42.1",
+    "cspell": "^6.10.1",
+    "eslint": "^8.27.0",
+    "eslint-config-prettier": "^8.5.0",
+    "eslint-config-vacuumlabs": "^8.0.2",
+    "eslint-plugin-import": "^2.25.4",
     "eslint-plugin-mocha": "^8.0.0",
     "eslint-plugin-simple-import-sort": "^7.0.0",
-    "mocha": "^9.2.0",
+    "mocha": "^10.2.0",
     "pkg": "^5.8.0",
-    "ts-node": "^9.0.0",
-    "typescript": "^4.0.3"
+    "prettier": "^2.6.2",
+    "ts-node": "^10.9.1",
+    "typescript": "^4.9.4"
   },
   "pkg": {
     "assets": "package.json"
   },
+  "engines": {
+    "node": "18.7.0",
+    "npm": ">=6.x",
+    "yarn": ">=1.21.3"
```

we actually just ran into issues on our project because Heroku when deploying tried to use yarn 2 which is not compatible with yarn 1, so it is safer to fix yarn major version to `1` and allow just a higher minor/patch version

```suggestion
    "yarn": "^1.21.3"
```